src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/PlaceholderLocalSymbol.cs

```diff
@@ -88,11 +88,43 @@ internal override EELocalSymbolBase ToOtherMethod(MethodSymbol method, TypeMap t
 
         internal static BoundExpression ConvertToLocalType(CSharpCompilation compilation, BoundExpression expr, TypeSymbol type, DiagnosticBag diagnostics)
         {
+            if (type.IsPointerType())
+            {
+                var syntax = expr.Syntax;
+                var intPtrType = compilation.GetSpecialType(SpecialType.System_IntPtr);
+                Binder.ReportUseSiteDiagnostics(intPtrType, diagnostics, syntax);
+                MethodSymbol conversionMethod;
+                if (Binder.TryGetSpecialTypeMember(compilation, SpecialMember.System_IntPtr__op_Explicit_ToPointer, syntax, diagnostics, out conversionMethod))
+                {
+                    var temp = ConvertToLocalTypeHelper(compilation, expr, intPtrType, diagnostics);
```

Comment:
Well, for one thing, I had never heard of ```System.Reflection.Pointer```.  For another, that type does not appear to be portable.

Currently, the debugger returns the raw pointer (per @pnelsonmsft).  I'm not sure what that means.  We're planning to change it to return a boxed ```IntPtr```.